yii

```diff
@@ -5,21 +5,8 @@ declare(strict_types=1);
 
 use App\Runner\ConsoleApplicationRunner;
 
-require_once __DIR__ . '/vendor/autoload.php';
-
-/**
- *  Set debug value for console application runner, for default its `true` add additionally the validation of the
- *  container-di configurations (debug mode).
- */
-define('YII_DEBUG', getenv('YII_DEBUG') ?: true);
-
-/**
- *  Set environment value for web application runner, for default its `null`.
- *
- *  @link https://github.com/yiisoft/config#environments
- */
-define('YII_ENV', getenv('YII_ENV') ?: null);
+require_once __DIR__ . '/preload.php';
 
 // Run console application runner
-$consoleRunner = new ConsoleApplicationRunner(YII_DEBUG, YII_ENV);
-$consoleRunner->run();
+$runner = new ConsoleApplicationRunner($_ENV['YII_DEBUG'], $_ENV['YII_ENV']);
```

Comment:
Shouldn't we use dotenv here as well as in `index.php`?

Reply:
Already used [here](https://github.com/yiisoft/app/pull/193/files#diff-f40671a7fbd0b0f3d9444229f7853a1b0924873bd9369552e313544bb6c6bb77R8).